### ISSUE TITLE
Localization Selector Uppercase with hyphen

### ISSUE THREAD

**Issue by alexander-schranz** [sulu-io/sulu#1074]
I would prefer e.g. `EN-US` in the localization selector

![bildschirmfoto 2015-05-07 um 13 27 50](https://cloud.githubusercontent.com/assets/1698337/7514361/f4c0fd3c-f4bc-11e4-9fc8-31e14698750a.png)

**Comment by danrot** [comment]
@berndhep @chirimoya Should we change this? Otherwise this issue should be closed.

**Comment by alexander-schranz** [comment]
ping @berndhep

**Comment by berndhep** [comment]
Upper cases would look bit more "pro". I give my thumbsup to @alexander-schranz input.

**Comment by alexander-schranz** [comment]
@danrot are they uppercase in 2.0?

**Comment by danrot** [comment]
Not yet :smiley:

**Comment by alexandersch** [comment]
I would prefer that it is translatable, so that we can show `English` for `EN-US` or `German` for `DE-DE`. And if your admin language is set to german you would see `Deutsch` and `Englisch`

**Comment by alexander-schranz** [comment]
We should then decide if things like `Shadow Locale` dropdown and the `User Role Locale` dropdown should then show this translation. I personally would prefer the generate the translation over the sulu Intl component to be consistent over all sulu installation. Because else this could get inconsistent if the developer can set the translation. We use the intl extension in the web-twig intl extension as example:

 - https://github.com/sulu/web-twig/blob/789a019e7472478e84f63bf1e43747808d603f10/src/IntlExtension.php#L170
 - https://github.com/sulu/web-twig/blob/789a019e7472478e84f63bf1e43747808d603f10/tests/Unit/IntlExtensionTest.php#L188-L189
  - https://github.com/sulu/web-twig/blob/789a019e7472478e84f63bf1e43747808d603f10/tests/Unit/IntlExtensionTest.php#L198-L199

**Comment by alexandersch** [comment]
I agree, automatic translations through Intl extension would be better than user translated strings. Maybe then also build in a possibility to configure which format to show. Alpha2, Alpha3 or full language name for example.
